src/Symfony/Component/Messenger/composer.json

```diff
@@ -20,6 +20,7 @@
         "psr/log": "~1.0"
     },
     "require-dev": {
+        "ext-json": "*",
```

Comment:
what does messenger require json for? also this is an unrelated changed.

Reply:
Because it uses `json_decode` and `json_encode`.
Yeah it is not related with this P.R., but it is so minimalist, I supposed it could pass.

Would you like I remove this commit?

Reply:
Yes. json is only used in certain cases like redis. So it's an optional dependency.